usaspending_api/disaster/v2/views/overview.py

```diff
@@ -105,3 +115,6 @@ def total_outlays(self):
             .aggregate(total=Sum("gross_outlay_amount_by_tas_cpe"))["total"]
             or 0.0
         )
+
+    def sum_values(self, obj, key):
```

Comment:
note for future work

Technically `obj` is a list of dictionaries, not an object. For a tiny function like this, I would probably make it a staticmethod, use slightly more descriptive argument names, and add type hints for additional communication. None of this is blocking the PR from being merged

Reply:
I went ahead and did this